src/ae.h

```diff
@@ -42,11 +42,11 @@
 #define AE_NONE 0     /* No events registered. */
 #define AE_READABLE 1 /* Fire when descriptor is readable. */
 #define AE_WRITABLE 2 /* Fire when descriptor is writable. */
-#define AE_BARRIER                                                                                                     \
-    4 /* With WRITABLE, never fire the event if the                                                                    \
-         READABLE event already fired in the same event                                                                \
-         loop iteration. Useful when you want to persist                                                               \
-         things to disk before sending replies, and want                                                               \
+#define AE_BARRIER                                       \
+    4 /* With WRITABLE, never fire the event if the      \
+         READABLE event already fired in the same event  \
+         loop iteration. Useful when you want to persist \
+         things to disk before sending replies, and want \
          to do that in a group fashion. */
```

Comment:
Would it reject it if we would do like this?

```suggestion
#define AE_BARRIER 4 /* With WRITABLE, never fire the event if the      \
                        READABLE event already fired in the same event  \
                        loop iteration. Useful when you want to persist \
                        things to disk before sending replies, and want \
                        to do that in a group fashion. */
```

I was most likely like this originally before clang-format changed it to the worse...

Reply:
Oops, that's my mistake, I've overlooked it. It will not reject it, I'll fix it :)